config/php-cs-fixer.php

```diff
@@ -21,6 +19,9 @@
             '@PHPUnit75Migration:risky' => true,
             '@PHPUnit84Migration:risky' => true,
 
+            // needed for PHP 7.x
+            'trailing_comma_in_multiline' => ['after_heredoc' => true, 'elements' => ['arrays', 'match']],
+
```

Comment:
I agree with the setting.  However, the comment may be misleading.  According to the [docs](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/rules/control_structure/trailing_comma_in_multiline.rst), the default is `['after_heredoc' => false, 'elements' => ['arrays']]`, which would not break PHP 7.2 (which doesn't allow trailing comma after last argument in function/method call) or PHP 7.x (which doesn't allow trailing comma after last parameter in function/method declaration).

Reply:
Good point! Done, rebased and repushed.

Reply:
What about the comment being misleading?  Really, the setting is including `match` which is a PHP >= 8.0 language construct (so won't be having any effect until we drop PHP 7.x and start using PHP 8 language constructs, however will be in place ready for such time).

Reply:
Done (hopefully).